classes/plugins/PluginHelper.inc.php

```diff
@@ -59,7 +65,7 @@ function extractPlugin($filePath, $originalFileName, &$errorMsg) {
 		// Test whether the tar binary is available for the export to work
 		$tarBinary = Config::getVar('cli', 'tar');
 		if (!empty($tarBinary) && file_exists($tarBinary)) {
-			exec($tarBinary.' -xzf ' . escapeshellarg($filePath) . ' -C ' . escapeshellarg($pluginExtractDir));
+			exec($tarBinary.' -xf ' . escapeshellarg($filePath) . ' -C ' . escapeshellarg($pluginExtractDir));
```

Comment:
The use of compression autodetection is specific to GNU tar and will fail on other tars, for example Solaris tar.